src/volue/mesh/_common.py

```diff
@@ -398,6 +398,41 @@ def _from_proto_curve_type(proto_curve: type.resources_pb2.Curve) -> Timeseries.
     return curve
 
 
+# FIXME: Don't have such duplicated functions.
```

Comment:
Agree. Can we do it in a separate PR together with other mapping functions like for curve types?